ivy/functional/backends/torch/experimental/losses.py

```diff
@@ -0,0 +1,22 @@
+
+# global
+import torch
+import torch.nn
+
+# local
+from ivy.func_wrapper import with_unsupported_dtypes
+from . import backend_version
+
+
+
+@with_unsupported_dtypes({"2.0.1 and below": "float"}, backend_version)
+def huber_loss(
+    true: torch.Tensor,
+    pred: torch.Tensor,
+    /,
+    delta: float = 1.0,
+    reduction: str = "mean",
+    out: torch.Tensor = None,
+) -> torch.Tensor:
+    huber_loss_fn = torch.nn.HuberLoss(reduction=reduction, delta=delta)
```

Comment:
You should be using `torch.nn.functional.huber_loss` here.Look at the documentation here https://pytorch.org/docs/stable/generated/torch.nn.functional.huber_loss.html and update the function call accordingly.